cni/network/network.go

```diff
@@ -653,10 +657,9 @@ func (plugin *NetPlugin) createNetworkInternal(
 		ServiceCidrs:                  ipamAddConfig.nwCfg.ServiceCidrs,
 	}
 
+	constructNetworkInfo(ipamAddResult, podSubnetPrefix, podSubnetV6Prefix, nwInfo)
```

Comment:
the func name should indicate that it is adding the subnets to networkInfo. The current name of the func doesn't give any idea on what is being contructed